dali/operators/decoder/host/fused/host_decoder_random_crop.cc

```diff
@@ -20,9 +20,30 @@
 #include "dali/image/image_factory.h"
 #include "dali/operators/decoder/host/fused/host_decoder_random_crop.h"
 #include "dali/pipeline/operator/common.h"
+#include "dali/pipeline/operator/checkpointing/snapshot_serializer.h"
 
 namespace dali {
 
+void HostDecoderRandomCrop::SaveState(OpCheckpoint &cpt, std::optional<cudaStream_t> stream) {
```

Comment:
As a follow-up - use AccessOrder instead of `optional<cudaStream_t>`?

Reply:
Sure